cg/store/api/delete.py

```diff
@@ -1,17 +1,31 @@
 """Handler to delete data objects"""
 
-from cg.store import models
+from typing import List
+from cg.store.models import Flowcell, Family, FamilySample, Sample
 from cg.store.api.base import BaseHandler
 
 
 class DeleteDataHandler(BaseHandler):
     """Contains methods to delete business data model instances"""
 
-    def delete_flowcell(self, flowcell_name: str):
-        flowcell: models.Flowcell = self.Flowcell.query.filter(
-            models.Flowcell.name == flowcell_name
-        ).first()
+    def delete_flowcell(self, flowcell_name: str) -> None:
+        flowcell: Flowcell = self.Flowcell.query.filter(Flowcell.name == flowcell_name).first()
         if flowcell:
             flowcell.delete()
             flowcell.flush()
             self.commit()
+
+    def delete_relationships_sample(self, sample: Sample) -> None:
+        """Delete relationships between all cases and the provided sample."""
+        if sample and sample.links:
+            for case_sample in sample.links:
+                case_sample.delete()
+            self.commit()
+
+    def delete_cases_without_samples(self, case_ids: List[str]) -> None:
+        """Delete any cases specified in case_ids without samples."""
+        for case_id in case_ids:
+            case: Family = self.Family.query.filter(Family.internal_id == case_id).first()

```

Comment:
Same question here, not sure how to access the methods in the find business data handler.

Reply:
Hm ok, We should make it so that this can access one API that can get data from the database. And there should only be one API/handlar doing it so that we do not duplicate functionality. But this might make the required changes go out of scope of this PR. If you cannot find an easy solution to getting access to the handler we should go with this and revisit in a focused PR to address this

Reply:
I could not find an easy way, lets leave it for another PR 👍